src/core/src/pso.rs

```diff
@@ -208,7 +208,7 @@ impl Descriptor {
 }
 
 /// A complete set of vertex buffers to be used for vertex import in PSO.
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
are these `core` changes required for your PR?
We need to bump the version numbers and (if core is changed) change the dependency of core to the new version.